sdk/core/Azure.Core/tests/TransportFunctionalTests.cs

```diff
@@ -94,7 +94,7 @@ public async Task SettingHeaderOverridesDefaultContentLength()
         public async Task CanSetContentLengthOverMaxInt()
         {
             long contentLength = 0;
-            using TestServer testServer = new TestServer(
+            TestServer testServer = new TestServer(
```

Comment:
disposing the TestServer for these two tests adds 4 seconds to the test. Gave up trying to figure out why.